config/routes.rb

```diff
@@ -1,5 +1,13 @@
 Rails.application.routes.draw do
 
+  get 'uploads/new'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.